### PR TITLE
Edit gitignore & add task 2 (Name-Adress)

### PR DESCRIPTION
**Pull request by forintbox**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,14 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
+# IntelliJ_IDEA
```
Comment (Hmeliumm) [inline .gitignore:13]: You dont't need ignore both .idea and **/.idea and other simillar ignore. Please leave only relevalent

File: TaskAdressProject/src/AdressPJ.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class AdressPJ {
```
Comment (Hmeliumm) [inline TaskAdressProject/src/AdressPJ.java]: What is PJ?

File: TaskAdressProject/src/AdressPJ.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    public static void main(String[] args) {
+        String name, address;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("What is your name?");
+        name = sc.nextLine();
+        System.out.println("Where are you live?, " + name);
+        address = sc.nextLine();
+        System.out.print("OK, you are:\n " + name + " from " + address);
+
```
Comment (Hmeliumm) [inline TaskAdressProject/src/AdressPJ.java]: Please remove extra line

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: TaskAdressProject/src/AdressPJ.java (hunk 3 above)
Comment (forintbox) [reply TaskAdressProject/src/AdressPJ.java]: Done!

File: .gitignore (hunk 1 above)
Comment (forintbox) [reply .gitignore:13]: Done!

File: TaskAdressProject/src/AdressPJ.java (hunk 3 above)
Comment (forintbox) [reply TaskAdressProject/src/AdressPJ.java]: This is the name of the project on PC and I mistakenly used it here :)
I already changed it

Comment (forintbox) [reply TaskAdressProject/src/AdressPJ.java]: This is the name of the project on PC and I mistakenly used it here :)
I already changed it

File: TaskAdressProject/src/AdressPJ.java (hunk 2 above)
Comment (forintbox) [reply TaskAdressProject/src/AdressPJ.java]: This is the name of the project on PC and I mistakenly used it here :)
I already changed it

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**